src/layouts/EditNavBar.jsx

```diff
@@ -612,25 +621,47 @@ const EditNavBar = ({ match }) => {
 
   return (
     <>
-      {showDeletedText && !isEmpty(deletedLinks) && (
+      {!isEmpty(deletedLinks) && (
```

Comment:
we should aim to inline this into the `isOpen` prop

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/513eedac744b2fe3c2814aff9b935982147c46d2